client/src/hooks/filters/index.js

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import usePriceFilter from "./usePriceFilter";
+
+export default function useFilters() {
+  const { matchesPriceCategory, PriceFilterComponent } = usePriceFilter();
```

Comment:
I think I have a rough idea of what's going on here, could you clarify exactly what's happening exactly?

Reply:
I made a custom hook off that returns a function and a component.

The function is the Boolean function that checks if a restaurant matches the current price category

The component is manav's price filter view


Similar to how useState returns a variable and a function to modify the variable,
I created a hook that returns
1. The matching condition
2. The component to render

Reply:
Ah, thank you!